docker/docker-compose.ltc.yml

```diff
@@ -0,0 +1,55 @@
+version: '2'
+services:
+    # ltc is an image of litecoin node which used as base image for ltcd and
+    # ltcctl. The environment variables default values determined on stage of
+    # container start within starting script.
+    ltcd:
+      image: ltcd
+      container_name: ltcd
+      build:
+        context: ltcd/
+      volumes:
+        - shared:/rpc
+        - litecoin:/data
+      environment:
+        - RPCUSER
+        - RPCPASS
+        - NETWORK
+        - DEBUG
+        - MINING_ADDRESS
+      entrypoint: ["./start-ltcd.sh"]
+
+    lnd:
+      image: lnd
+      container_name: lnd_ltc
+      build:
+        context: ../
+        dockerfile: docker/lnd/Dockerfile
+      environment:
+        - RPCUSER
+        - RPCPASS
+        - NETWORK
+        - CHAIN
+        - DEBUG
+      volumes:
+        - shared:/rpc
+        - lnd:/root/.lnd
+      entrypoint: ["./start-lnd.sh"]
+      links:
```

Comment:
Should we remove links, which is deprecated?

> Links are a legacy option. We recommend using networks instead.

Reply:
@johng removing the legacy issues could take place in a separate PR.  I think there are enough changes in this PR and better to keep the PR simple and address the github issue imo.